tests/utils.c

```diff
@@ -177,6 +177,15 @@ test_device_get_partition_path_fail(void)
     g_assert_null(path);
 }
 
+static void
+test_get_file_size(void)
+{
+    g_autoptr(GError) error = NULL;
+    goffset size = pu_get_file_size("data/root.ext4", &error);
+    g_assert_no_error(error);
+    g_assert_cmpuint(size, ==, 262144);
```

Comment:
Better make the EXT4 image size a `#define`, so it is easier to be found, identified and changed in the future.